huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -522,6 +527,10 @@ func resourceRdsInstanceCreate(ctx context.Context, d *schema.ResourceData, meta
 		}
 	}
 
+	if err := updateRdsInstanceBackupStrategy(d, client, instanceID); err != nil {
```

Comment:
Would this function call be correct responding when no `backup strategy` hcl block there?